mxcube3/core/components/queue.py

```diff
@@ -2294,6 +2301,18 @@ def add_centring(self, _id, params):
 
         return {"QueueId": new_node, "Type": "Centring", "Params": params}
 
+    def update_dependent_field(self, task_name, data):
```

Comment:
what is this for? What is `task_name` here? `task_name` for me would be `DataCollection`...

Reply:
:), I guess this one is fine as we spoke about this during our meeting today,